src/app/styles/_constants.scss

```diff
@@ -34,6 +34,18 @@ $color-red: hsl(6deg 65% 52%); // #d44333
 $color-purple-400: hsl(257deg 45% 54%); // #7356bf
 $color-purple-50: hsl(260deg 47% 96%); // #f4f1fa
 
+// Technologies theme colors
```

Comment:
Why we need all these new variables?

Also the comment doesn't make any sense to me.

Reply:
At the moment these colors are hardcoded in the `.data` file and if you need to use them somewhere else, it isn't very easy to find them, so I moved them to variables so that they are fixed. The comment was suggested by Ilya, I couldn't think of a better one.